java/wavefrontsender.md

```diff
@@ -5,103 +5,158 @@
+```
+
+**Example**: Use a builder to create a WavefrontClient and send  data to Wavefront via direct ingestion.
 
 ```java
-// Create a builder with the Wavefront URL and a Wavefront API token
-// that was created with direct ingestion permission.
-WavefrontDirectIngestionClient.Builder wfDirectIngestionClientBuilder =
-  new WavefrontDirectIngestionClient.Builder(wavefrontURL, token);
+// Using the WavefrontClient.Builder directly with a url in the form of "https://DOMAIN.wavefront.com"
+// and a Wavefront API token with direct ingestion permission
+WavefrontClient.Builder wfClientBuilder = new WavefrontClient.Builder(wavefrontURL, token);
+
+//The maximum message size in bytes that is pushed with on each flush interval 
+wfClientBuilder.messageSizeBytes(120);
 
-// Optional: Override the max queue size (in data points). Default: 50,000
-wfDirectIngestionClientBuilder.maxQueueSize(100_000);
+// This is the size of internal buffer beyond which data is dropped
+// Optional: Set this to override the default max queue size of 50,000
+wfClientBuilder.maxQueueSize(100_000);
 
-// Optional: Override the batch size (in data points). Default: 10,000
-wfDirectIngestionClientBuilder.batchSize(20_000);
+// This is the max batch of data sent per flush interval
+// Optional: Set this to override the default batch size of 10,000
+wfClientBuilder.batchSize(20_000);
 
-// Optional: Override the flush interval (in seconds). Default: 1 second
-wfDirectIngestionClientBuilder.flushIntervalSeconds(2);
+// Together with batch size controls the max theoretical throughput of the sender
+// Optional: Set this to override the default flush interval value of 1 second
+wfClientBuilder.flushIntervalSeconds(2);
 
-// Create a WavefrontDirectIngestionClient.
-WavefrontSender wavefrontSender = wfDirectIngestionClientBuilder.build();
- ```
+WavefrontSender wavefrontSender = wfClientBuilder.build();
+``` 
+
+## Send Data to Multiple Wavefront Services
+
+Use `WavefrontMultiClient` to send data to multiple Wavefront services so you handle the data traffic.
+The `addClient()` supports null for batch size, queue size, and push interval. The defaults values are used if nothing is specified.
+
+**Example**: Creating a `WavefrontMultiClient` to send data to multiple Wavefront services.
+```java
+// Add multiple URLs to the Factory to obtain a multi-sender
+WavefrontClientFactory wavefrontClientFactory = new WavefrontClientFactory();
+wavefrontClientFactory.addClient("https://someToken@DOMAIN.wavefront.com");
+wavefrontClientFactory.addClient("proxy://our.proxy.lb.com:2878");
+// Send traces and spans to the tracing port. If you are directly using the sender SDK to send spans without using any other SDK, use the same port as the customTracingListenerPorts configured in the wavefront proxy. Assume you have installed and started the proxy on <proxyHostname>.
+wavefrontClientFactory.addClient("http://<proxy_hostname>:30000/");
```

Comment:
I feel like this is a source of confusion, especially now that you can send tracing data to 2878 (in proxy 7.0+ I believe).  The only time you would want to do something like this is to send to the custom tracing port. So might be best to change this example to send to 30001 and the comment to make clear this should only be included when using the sender SDK to send spans (and maybe add a link to documentation about the custom tracing port)